test/Microsoft.ML.Tests/TrainerEstimators/SdcaTests.cs

```diff
@@ -140,13 +141,15 @@ public void SdcaLogisticRegressionWithWeight()
             bool sameScores = true;
             for (int i = 0; i < scores1.Length; i++)
             {
-                if(!CompareNumbersWithTolerance(scores1[i], scores2[i]))
+                if(!CompareNumbersWithTolerance(scores1[i], scores2[i], allowMismatch: true))
                 {
```

Comment:
Do we need allowMismatch or will specifying the digitsOfPrecision work?

Reply:
We need this allowMismatch, these test are intend to get the conclusion of these 2 scores are different.

---
In reply to: [375022988](https://github.com/dotnet/machinelearning/pull/4782#discussion_r375022988) [](ancestors = 375022988)